### PR TITLE
Added commentary about base_url port requirements

### PR DESCRIPTION
**Pull request by agrieco**
Update based on learnings of:
https://github.com/home-assistant/home-assistant/issues/29303

**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by frenck** [comment]
Hi there, @agrieco!

Thank you for the PR, however, I'm going to friendly decline.

The problem is: This is not just SmartThings, and it really depends on the integration.

This should be added to the documentation of that specific integration (in this case SmartThings).

**Comment by agrieco** [comment]
> Hi there, @agrieco!
> 
> Thank you for the PR, however, I'm going to friendly decline.
> 
> The problem is: This is not just SmartThings, and it really depends on the integration.
> 
> This should be added to the documentation of that specific integration (in this case SmartThings).

I had thought that was the page I edited- the specific integration steps for ST?

Either way, can we please update the appropriate docs- I spent hours upon hours figuring this out and from what I can tell, its not documented anywhere?

File: source/_integrations/smartthings.markdown [hunk 1]
```diff
@@ -46,7 +46,7 @@ If you are using Home Assistant Cloud (Nabu Casa) the integration will create a
 
 1. A [personal access token](https://account.smartthings.com/tokens) tied to a Samsung or SmartThings account (see below for instructions).
 2. Home Assistant setup for [remote access](/docs/configuration/remote/) via a domain name secured with SSL. *Self-signed SSL certificates are not supported by the SmartThings Cloud API.*
-3. [`base_url` of the http component](/integrations/http#base_url) set the URL that Home Assistant is available on the internet.
+3. [`base_url` of the http component](/integrations/http#base_url) set the URL that Home Assistant is available on the internet.  SmartThings requries this base_url to use a standard port (443).
```
Comment (frenck) [inline source/_integrations/smartthings.markdown]:
```suggestion
3. [`base_url` of the http integration](/integrations/http#base_url) set the URL that Home Assistant is available on the internet. SmartThings requires the `base_url` and Home Assistant to use the standard HTTPS port (443).
```

**Review by frenck — CHANGES_REQUESTED** [review]
You are totally right @agrieco! My bad, I overlooked that.

I've done a suggestion for a minor improvement. If you commit that suggestion, I would say: Good to go!

**Review by frenck — APPROVED** [review]
Awesome! 👍

**Pull request merged**